model/pod/db_test.go

```diff
@@ -1 +1,41 @@
 package pod
+
+import (
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/utility"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFindByInitializing(t *testing.T) {
+	require.NoError(t, db.Clear(Collection))
+
+	p := &Pod{
+		ID:     utility.RandomString(),
+		Status: StatusInitializing,
+		Secret: "secret",
```

Comment:
I don't think you need to set the secret since it doesn't have any relation to the query being tested.